src/js/popup.js

```diff
@@ -777,9 +777,20 @@ Logic.registerPanel(P_CONTAINER_INFO, {
       tr.classList.add("container-info-tab-row");
       tr.innerHTML = escaped`
         <td></td>
-        <td class="container-info-tab-title truncate-text" title="${tab.url}" >${tab.title}</td>`;
+        <td class="container-info-tab-title truncate-text" title="${tab.url}" >${tab.title}</td>
+        <td><img src="/img/container-close-tab.svg" style="width:50%;" id="close-tab" class="container-close-tab" /></td>`;
```

Comment:
Please put all styles in the CSS rather than in the HTML. I think it would be better if the X is only visible on hover that way the title can be longer until the user hovers.

Reply:
The only reason of putting inline CSS here is because width property seems to be blacklisted by lint, any other way to resize the image? And, yes X is only visible when someone hovers over the row.